src/types.ts

```diff
@@ -12,3 +12,10 @@ export type LessonResource = Resource & {
   id: string | number
   tags: any[]
 }
+
+export type Lesson = {
```

Comment:
We will chat about this, and what gets added to types in general. You can tell from the signature below that something isn't coded properly because of its complexity.